package com.company.day15.exceptionDemo;

/**
 * @author Joker_Dong
 * @date 2020-7-30 11:03
 */

public class Demo01 {
    public static void main(String[] args) {
        int[] arr = {1,2,3};
//        System.out.println(arr[3]);
        if (1==1){
//            throw new NullPointerException("要访问的arr数组不存在");
//            throw new ArrayIndexOutOfBoundsException("数组越界了，兄弟");
        }
    }
}
